external-import/wiz/src/external_import_connector/connector.py

```diff
@@ -0,0 +1,199 @@
+        self.converter_to_stix = ConverterToStix(self.helper)
+
+    def _collect_intelligence(self) -> list:
+        """
+        Collect intelligence from the source and convert into STIX object
+        :return: List of STIX objects
+        """
+
+        # ===========================
+        # === Add your code below ===
+        # ===========================
+
+        # Get entities from external sources
+        entities = self.client.get_entities()["objects"]
+        state = self.helper.get_state()
+        i = 0
+
+        for entity in entities:
+
+            # Filter entities
+            if "modified" in entity and state is not None:
+                if entity["modified"] < state["last_run"]:
+                    del entities[i]
+                    continue
+            i += 1
+
+        return entities
+
+        # ===========================
+        # === Add your code above ===
+        # ===========================
+
+    def process_message(self) -> None:
+        """
+        Connector main process to collect intelligence
+        :return: None
+        """
+        self.helper.connector_logger.info(
+            "[CONNECTOR] Starting connector...",
+            {"connector_name": self.helper.connect_name},
+        )
+
+        try:
+            # Get the current state
+            now = datetime.now()
+            current_timestamp = int(datetime.timestamp(now))
+
+            current_state = self.helper.get_state()
+
+            if current_state is not None and "last_run" in current_state:
+                last_run = current_state["last_run"]
+
+                self.helper.connector_logger.info(
+                    "[CONNECTOR] Connector last run",
+                    {"last_run_datetime": last_run},
+                )
+            else:
+                self.helper.connector_logger.info(
+                    "[CONNECTOR] Connector has never run..."
+                )
+
+            # Friendly name will be displayed on OpenCTI platform
+            friendly_name = "WIZ Threat Landscape"
+
+            # Initiate a new work
+            work_id = self.helper.api.work.initiate_work(
+                self.helper.connect_id, friendly_name
+            )
+
+            self.helper.connector_logger.info(
+                "[CONNECTOR] Running connector...",
+                {"connector_name": self.helper.connect_name},
+            )
+
+            # Performing the collection of intelligence
+            # ===========================
+            # === Add your code below ===
+            # ===========================
+            stix_objects = self._collect_intelligence()
+
+            if stix_objects is not None and len(stix_objects) is not None:
+                stix_objects_bundle = self.helper.stix2_create_bundle(stix_objects)
+                bundles_sent = self.helper.send_stix2_bundle(
+                    stix_objects_bundle, update=True
+                )
```

Comment:
@Renizmy Ok, I see. You need to add the work ID that you have initiated before to allow OpenCTI to identify the job to be processed

```suggestion
                bundles_sent = self.helper.send_stix2_bundle(
                    stix_objects_bundle, 
                    work_id=work_id
                    update=True
                )
```